### ISSUE TITLE
Verify documentation for CustomExecution API

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode#83435]
Ref: https://github.com/microsoft/vscode/issues/80375

- [ ] anyOS
- [ ] anyOS

Complexity: 2

The API was already tested [here](https://github.com/microsoft/vscode/issues/71089) so for this item:
- Verify that the documentation in the API makes sense
- Verify that the sample works as is and makes sense: https://github.com/microsoft/vscode-extension-samples/tree/master/task-provider-sample/

**Comment by bpasero** [comment]
@alexr00 running the sample on a small workspace I see this and the picker never completes:

![image](https://user-images.githubusercontent.com/900690/67760457-a89a1800-fa41-11e9-96ba-758e85381a20.png)

**Comment by alexr00** [comment]
@bpasero did you do an npm install? I had to bump the version of vscode and I saw the same thing.

**Comment by bpasero** [comment]
Yeah I did what the readme told me: `npm install` and `npm run compile`

**Comment by alexr00** [comment]
Can you try starting with a `git clean -xfd`?
